mozilla-release/browser/components/nsBrowserGlue.js

```diff
@@ -2301,6 +2301,9 @@ BrowserGlue.prototype = {
       // This is a new profile, nothing to migrate.
       Services.prefs.setIntPref("browser.migration.version", UI_VERSION);
 
+      // CLIQZ-SPECIAL: Default disabled AFT for new users.
+      Services.prefs.setIntPref("browser.privatebrowsing.apt", false);
```

Comment:
@tsl143 Are you sure it works? Shouldn't it be Services.prefs.setBoolPref(...) instead?

Reply:
Already updated :P

Reply:
@tsl143 great, thanks!